src/analyses/accessAnalysis.ml

```diff
@@ -105,7 +105,10 @@ struct
   let enter ctx lv f args : (D.t * D.t) list =
     [(ctx.local,ctx.local)]
 
-  let combine ctx lv fexp f args fc al f_ask =
+  let combine_env ctx lval fexp f args fc au f_ask =
+    ctx.local
+
+  let combine_assign ctx lv fexp f args fc al f_ask =
     access_one_top ctx Read false fexp;
```

Comment:
Is this correct if `fexp` is modified inside `f`? I would think this needs to happen before? Potentially even in enter?

Reply:
It's only read. I thought about moving these to `enter` but it doesn't have the original function expression argument, only the resolved name. It could be changed but I don't want to change every `enter` here as well. It's been fine since forever.

Reply:
Still, the access should be in `combine_env` right?

Reply:
I'll move it there to be slightly closer to where it should be, although it wouldn't still be fully precise.